### ISSUE TITLE
[Question]: Facing trouble with mmpbsa sander and PB calculation for protein-ligand complex

### ISSUE THREAD

**Issue by Rajath701** [Valdes-Tresanco-MS/gmx_MMPBSA#503]
### My Question is...

[INFO   ]   calculating complex contribution...              53%|#########################2                      | 316/600 [elapsed: 1:49:59 remaining: 3:22:20]
[ERROR  ] CalcError  
/home/rajath/miniconda3/envs/gmxMMPBSA/bin/sander failed with prmtop COM.prmtop!  
If you are using sander and PB calculation, check the *.mdout files to get the sander error   

Check the gmx_MMPBSA.log file to report the problem.   
File "/home/rajath/miniconda3/envs/gmxMMPBSA/bin/gmx_MMPBSA", line 8, in <module>  
 sys.exit(gmxmmpbsa())   
File "/home/rajath/miniconda3/envs/gmxMMPBSA/lib/python3.10/site-packages/GMXMMPBSA/app.py", line 101, in gmxmmpbsa  
 app.run_mmpbsa()  
File "/home/rajath/miniconda3/envs/gmxMMPBSA/lib/python3.10/site-packages/GMXMMPBSA/main.py", line 205, in run_mmpbsa    
self.calc_list.run(rank, self.stdout)   
File "/home/rajath/miniconda3/envs/gmxMMPBSA/lib/python3.10/site-packages/GMXMMPBSA/calculation.py", line 142, in run     
calc.run(rank, stdout=stdout, stderr=stderr)  
 File "/home/rajath/miniconda3/envs/gmxMMPBSA/lib/python3.10/site-packages/GMXMMPBSA/calculation.py", line 625, in run     
GMXMMPBSA_ERROR('%s failed with prmtop %s!\n\t' % (self.program, self.prmtop) +   
File "/home/rajath/miniconda3/envs/gmxMMPBSA/lib/python3.10/site-packages/GMXMMPBSA/exceptions.py", line 171, in __init__    
 raise exc('\n\n' + msg + '\n\nCheck the gmx_MMPBSA.log file to report the problem.') 
CalcError:  
/home/rajath/miniconda3/envs/gmxMMPBSA/bin/sander failed with prmtop COM.prmtop!
If you are using sander and PB calculation, check the *.mdout files to get the sander error
Check the gmx_MMPBSA.log file to report the problem.
Exiting. All files have been retained.

**Comment by marioernestovaldes** [comment]
could you please send the *.mdout files ?

**Comment by Rajath701** [comment]
[gmx_MMPBSA.log](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/15411335/gmx_MMPBSA.log)

**Comment by marioernestovaldes** [comment]
this is not a mdout file... there should be something liek this in the folder:

_GMXMMPBSA_complex_pb.mdout

**Comment by Rajath701** [comment]
[_GMXMMPBSA_complex_pb.mdout.0.txt](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/15411612/_GMXMMPBSA_complex_pb.mdout.0.txt)

**Comment by marioernestovaldes** [comment]
I don't see any issue, could you please send the files you are using?

**Comment by Rajath701** [comment]
> I don't see any issue, could you please send the files you are using?

I would like to add one more point: Could this problem be caused by high RAM usage?

**Comment by Rajath701** [comment]
[mmpbsa.in.txt](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/15412995/mmpbsa.in.txt)
[topol.txt](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/15412996/topol.txt)

If you need any file specifically please let me know.

**Comment by marioernestovaldes** [comment]
all the files you used to run the calcualtions

**Comment by Rajath701** [comment]
https://www.dropbox.com/scl/fo/805j41bikdaspb2izdis5/ADejyo5selFDHf59lyZGQK8?rlkey=o0z7y6xnksbqztbg2z50sk7it&st=eefvz4mx&dl=0

I hope you can access the link.
Thank you.

**Comment by marioernestovaldes** [comment]
ould you please sned the force field folder?

**Comment by Rajath701** [comment]
> ould you please sned the force field folder?

Yes, sure. I have added the force field folder to the dropbox now.

**Comment by marioernestovaldes** [comment]
I see in the trajectory there are some frames where the ligand is not in the binding site, I don't know if it is either pBC conditions or if the ligand just is unbound. Anyway, my recommendation is to check thoroughly the trajectory before attempting any calculation as this is probably the cause of the error

**Comment by Rajath701** [comment]
> I see in the trajectory there are some frames where the ligand is not in the binding site, I don't know if it is either pBC conditions or if the ligand just is unbound. Anyway, my recommendation is to check thoroughly the trajectory before attempting any calculation as this is probably the cause of the error

Thank you very much for your suggestion. I have analyzed the output of simulation and found to be the same. I will restrain the ligand at the binding site and reproduce the md trajectories.
Once again, thanks for your timely support!
